modules/solid_mechanics/src/materials/ADComputeShellStress.C

```diff
@@ -38,12 +38,16 @@ ADComputeShellStress::ADComputeShellStress(const InputParameters & parameters)
   std::unique_ptr<QGauss> t_qrule = std::make_unique<QGauss>(
       1, Utility::string_to_enum<Order>(getParam<std::string>("through_thickness_order")));
   _t_points = t_qrule->get_points();
+  _t_weights = t_qrule->get_weights();
   _elasticity_tensor.resize(_t_points.size());
   _stress.resize(_t_points.size());
   _stress_old.resize(_t_points.size());
   _strain_increment.resize(_t_points.size());
+  _local_transformation_matrix.resize(_t_points.size());
   _covariant_transformation_matrix.resize(_t_points.size());
   _global_stress.resize(_t_points.size());
+  _local_stress.resize(_t_points.size());
+  _t_shell = &getMaterialProperty<Real>("shell_thickness");
```

Comment:
As I noted for the strain calculator class, I think it's better to directly couple in the shell thickness as a variable here rather than computing it as a material property in the other class and retrieving it here.

Reply:
done